app/src/main/java/com/example/capstoneproject/model/api/BitsoResponse.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.capstoneproject.model.api
+
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+
+data class BitsoResponse(
+    @Expose @SerializedName("success") val success: Boolean,
```

Comment:
I'm wondering if this Expose annotation is being used for a specific case? :D

Reply:
It's not used, I was following an internet example and didn't get if it was needed. I just check https://stackoverflow.com/questions/34752200/gson-expose-vs-serializedname and understood I don't need it